Prox/Prox/Utilities/PlaceUtilities.swift

```diff
@@ -63,9 +63,7 @@ struct PlaceUtilities {
     static func updateReviewUI(fromProvider provider: PlaceProvider?, onView view: ReviewContainerView, isTextShortened: Bool = false) {
         guard let provider = provider,
                 !(provider.totalReviewCount == 0 && provider.rating == nil) else { // intentional: if both null, short-circuit
-            setSubviewAlpha(0.4, forParent: view)
```

Comment:
What if 1 provider has data but the other does not? Do we really want to one provider's collapse the UI in that case?